elastic/security/workflows/network/11.json

```diff
@@ -582,7 +582,11 @@
                 "include_unmapped": true
               },
               {
+                {% if index_mode != "logsdb" %}
```

Comment:
It looks like our template engine does not process files in the `workflows` directory. As a result we are not able to conditionally execute the query on `message` or `kubernetes.even.message`. I will just unconditionally execute it using `kubernetes.even.message` that is the original field name. After we introduce support for `copy_to` we can restore the original behaviour.